ososedki_dl/crawlers/ososedki_crawler.py

```
@@ -184,16 +207,18 @@ async def _extract_paginated_images(
 
         while True:
             try:
-                response = await self.session.post(url, json=payload, timeout=MAX_TIMEOUT)
-                response.raise_for_status()
-                response_json: dict[str, Any] = await response.json()
+                data = await self.downloader.fetch(
+                    url, "POST", response_property="json", json=payload
+                )
             except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)